analyzers/tests/SonarAnalyzer.UnitTest/TestCases/GetHashCodeEqualsOverride.CSharp9.cs

```diff
@@ -9,5 +9,5 @@ public record Noncompliant
 {
     private readonly int x;
 
-    public override int GetHashCode() => x.GetHashCode() ^ base.GetHashCode(); //Noncompliant - FP, this rule should not apply for records since Equals and GetHashCode are value-based
```

Comment:
Please rename the record as it is not noncompliant and also leave there a comment for the future to know why it is compliant.
Also it would be nice to verify that we work with Positional records.

Reply:
For this rule all records are ignored. It does not matter if they are positional or not.

Reply:
I know, I just wanted test cases to have a bigger safety net.